.eslintrc.js

```diff
@@ -31,16 +36,10 @@ module.exports = {
    */
   rules: {
     'consistent-this': ['error', 'self'],
-    'linebreak-style': 'off', // Doesn't play nicely with Windows
-    // just as bad as "max components per file"
-    'max-classes-per-file': 'off',
-    'no-alert': 'error',
-    // Strict, airbnb is using warn; allow warn and error for dev environments
-    'no-console': ['error', { allow: ['warn', 'error'] }],
-    'no-constant-condition': 'error',
-    // Airbnb use error
-    'no-param-reassign': 'off',
-    'no-prototype-builtins': 'off',
+    'max-classes-per-file': 'off', // just as bad as "max components per file"
```

Comment:
nit: keep this above the rule to preserve git history. For longer explanation we need to move it above the line anyway.